dc/index.html

```diff
@@ -346,4 +476,18 @@
 	</div>
 
 </body>
+
+<script src="/js/vendor/jquery-3.2.1.min.js" type="text/javascript"></script>
+<script type="text/javascript">
+(function($){
+	"use strict";
+	$(document).ready(function() {
+		$(document).on("can_embed_submitted", function(evt) {
+			$("#petition_thank_you").fadeIn();
```

Comment:
Could pretty easily do this by setting a `.fade { transition: 1s; }` and `document.getElementById('petition_thank_you').classList.remove('hidden', 'invisible')` on `.hidden { display: none; }` and `.invisible { opacity: 0; }` instead of including all of jQuery.